### DISCUSSION TITLE
pytorch-forecasting installation on databricks notebook

### DISCUSSION THREAD

**Discussion by MuhammadIrfanAli** [jdb78/pytorch-forecasting#1687] [General]
Hi,

I am trying to install the library on databricks notebook but I keep hitting conflict issues with  `typing_extensions`.

Tried to use version 4.8.0 as well but still cannot import the library. Does anyone else has faced this issue before. If someone knows the compatible version, this would be great.

`%pip install pytorch-lightning==2.1.1 pytorch-forecasting typing-extensions==4.8.0`

**Comment by fkiraly** [comment]
could you tell us your python version and your `pip freeze` output?
